src/fde.h

```diff
@@ -50,6 +50,9 @@ class fde
 {
 
 public:
+
+    static void CreateTable();
```

Comment:
After looking at some of the callers, I think it would be better to call this method Init() -- it initializes the low-level descriptor management module, and we have many other similar methods called Init() or functions called FooInit(). I do not insist on this change though. Your call.